package.json

```diff
@@ -66,17 +66,15 @@
     "webpack-manifest-plugin": "1.3.2",
     "whatwg-fetch": "2.0.3"
   },
-  "config": {
-    "deploymentNetwork": "ropsten"
-  },
+
   "scripts": {
     "start": "node scripts/start.js",
     "build": "node scripts/build.js",
     "test": "run-s test:truffle 'test:app --all'",
     "test:ci": "run-s build test:truffle 'test:app --all --ci'",
     "test:app": "node scripts/test.js --env=jsdom",
     "test:truffle": "node scripts/test_truffle.js",
-    "truffle:migrate": "node_modules/.bin/truffle migrate --network $npm_package_config_deploymentNetwork",
+    "truffle:migrate": "node -e 'require(\"./scripts/deploy_contracts.js\").deployContracts()'",
```

Comment:
What's the benefit of having a script that exports a function, then importing/calling that function in a second "script" via `node -e`?

For comparison, the rest of the scripts files are simply scripts that are run.

Reply:
I did this for ease of use of calling the logic from other scripts. The main use case of `deploy_contracts` is to be called from the `build` and `start` script.

 We need to call `deploy_contracts` from these scripts to ensure that `process.env.TARGET_NETWORK` has been setup correctly before attempting to deploy.

 It seemed more natural for this to be a function then a script that has to be invoked with a `child_process`.

The `truffle:migrate` command was just updated to maintain compatibility.

Perhaps this would be clearer if the `deploy_contracts.ts` was moved to a helper folder or file within the scripts folder.

Reply:
Makes sense.

It seems fine in the scripts folder, though it is perhaps a little out of place.